docker/compose.yml

```diff
@@ -0,0 +1,62 @@
+# See https://supertokens.com/docs/deployment/self-host-supertokens for details.
+
+name: grassroots
+services:
+  # Note: If you are assigning a custom name to your db service on the line below, make sure it does not contain underscores
+  db:
+    container_name: db
+    image: 'postgres:latest'
+    environment:
+      POSTGRES_USER: supertokens_user
+      POSTGRES_PASSWORD: somePassword
+      POSTGRES_DB: grassroots
+    ports:
+      - 5432:5432
+    networks:
+      - app_network
+    restart: unless-stopped
+    healthcheck:
+      test: ['CMD', 'pg_isready', '-U', 'supertokens_user', '-d', 'grassroots']
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  supertokens:
+    container_name: supertokens
```

Comment:
Super token looks really cool. I'd like to learn more about what we can or can't do at the free tier. I have dreams of setting up a unified sign-in experience for volunteers and wouldn't want to lock us into a platform that would make that prohibitively expensive.

Reply:
I _think_ the free tier mostly just means you need to host it yourself. It doesn't seem super constrained.

Though there was something about even when you're hosting it yourself, their "use management dashboard" is heavily restricted, to 3 users or something.

Definitely worth double checking.

Reply:
Oh, there are a bunch of things that cost money even when you self-host.
https://supertokens.com/pricing has the details.

I think you've got much more context than I do on how crucial those features are, and how reasonable the pricing is.